jboss-tck-runner/pom.xml

```diff
@@ -21,8 +21,10 @@
     <url>http://weld.cdi-spec.org</url>
 
     <properties>
-        <htmlunit.version>2.9</htmlunit.version>
-        <shrinkwrap.version>1.0.1</shrinkwrap.version>
+        <!-- This matches the htmlunit version in TCK -->
+        <htmlunit.version>2.50.0</htmlunit.version>
```

Comment:
This should be inherited from core POM, see https://github.com/weld/core/blob/master/pom.xml#L72
Therefore, this shouldn't be needed (probably a leftover from some temporary issues).

Reply:
The problem was that weld was using a much newer version of htmlunit than the TCK was, so it had been overriden to 2.9 at this level previously. I was expecting that these versions might diverge again in the future and an explicit override would still be needed here.